packages/ui/src/TapToEdit.tsx

```diff
@@ -9,70 +9,76 @@ import {Text} from "./Text";
 
 export interface TapToEditProps extends Omit<FieldProps, "onChange" | "value"> {
   title: string;
-  initialValue: any;
+  value: any;
+  setValue: (value: any) => void;
   // Not required if not editable.
   onSave?: (value: any) => void | Promise<void>;
   // Defaults to true
   editable?: boolean;
+  // enable edit mode from outside the component
+  editingOverride?: boolean;
```

Comment:
Can we do something like "isEditing" to match typical React props names? But still mention in the comments it overrides the internal state.

Reply:
that works! I struggled with what to name it.